src/support_sphere/lib/presentation/pages/main_app/profile/profile_body.dart

```diff
@@ -189,6 +132,57 @@ class _PersonalInformation extends StatelessWidget {
               ],
             ),
           ],
+          modalBody: FormBuilder(
+            key: formKey,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                FormBuilderTextField(
+                  name: 'givenName',
+                  decoration: const InputDecoration(labelText: 'Given Name'),
```

Comment:
Can you put these label texts into the string catalog constants? We'll need for all the hardcoded label text